x-pack/plugin/vector-tile/src/javaRestTest/java/org/elasticsearch/xpack/vectortile/VectorTileRestIT.java

```diff
@@ -235,7 +242,7 @@ public void testIndexAllGet() throws Exception {
         final VectorTile.Tile tile = execute(mvtRequest);
         assertThat(tile.getLayersCount(), Matchers.equalTo(3));
         // 33 points, 1 polygon and two from geometry collection
-        assertLayer(tile, HITS_LAYER, 4096, 36, 2);
+        assertLayer(tile, HITS_LAYER, 4096, 37, 2);
```

Comment:
Perhaps the comment one line up needs to be updated.

Reply:
good catch updated.